### PR TITLE
SOC-1088 | Fix for forgot password on CNW

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/SOC-1088

### CODE REVIEW

**Comment by garthwebb** [comment]
LGTM

File: extensions/wikia/UserLogin/js/UserLoginAjaxForm.js [hunk 1]
```diff
@@ -93,10 +93,14 @@
...
 			form = this.form;
 
-			form.slideUp(duration, function () {
+			form.slideUp(duration, (function () {
+				var newForm = content.find('form');
 				form.replaceWith(content);
 				content.slideDown(duration);
-			});
+				newForm.on('submit', this.submitLoginAfterResetPass.bind(this));
+				this.wikiaForm = new WikiaForm(newForm);
```
Comment (rafalkalinski) [inline extensions/wikia/UserLogin/js/UserLoginAjaxForm.js:101]: do we really need this property?

**Comment by rafalkalinski** [comment]
LGTM

File: extensions/wikia/UserLogin/js/UserLoginAjaxForm.js (hunk 1 above)
Comment (kvas-damian) [inline extensions/wikia/UserLogin/js/UserLoginAjaxForm.js:101]: While we are overriding initial form I wanted update all attributes to match current state.

**Pull request merged**